### PR TITLE
Docs: TypeScript Guide

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

File: docs/advanced-chart-features/highcharts-typescript-declarations.md [hunk 1]
```diff
@@ -220,14 +150,15 @@ as illustrated with the below example.
 
 ### Steps
 
-* Set proper types for your extensions
-* Declare additional interfaces with your extensions to the Highcharts namespace
+* User proper typing in your extensions
```
Comment (Xillez) [inline docs/advanced-chart-features/highcharts-typescript-declarations.md:153]: Typo: "User" => "Use"

File: docs/advanced-chart-features/highcharts-typescript-declarations.md [hunk 2]
```diff
@@ -292,3 +223,64 @@ Highcharts.Point.prototype.highlight = function (
     this.series.chart.xAxis[0].drawCrosshair(event, this);
 };
 ```
+
+Note: If you like to replace existing functionallity of Highcharts, listen to
```
Comment (Xillez) [inline docs/advanced-chart-features/highcharts-typescript-declarations.md:227]: Typo: "functionallity" => "functionality"

**Review by Xillez — CHANGES_REQUESTED** [review]
Just some typos as far as I can see 👍

**Review by Xillez — APPROVED** [review]
LGTM 👍

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Review by KacperMadej — APPROVED** [review]

**Pull request merged**